ejercicio1.rb

```diff
@@ -33,27 +30,24 @@
               }
             }
           }
-        
 
   http = Net::HTTP.new(uri.host, uri.port)
   http.use_ssl = true
   http.verify_mode = OpenSSL::SSL::VERIFY_NONE
 
   request = Net::HTTP::Post.new(uri.request_uri)
-  request['Authorization'] = 'token c90f2da8cea3cef00e96e96bc1433d7024479715'
-
+  request['Authorization'] = 'token bb3f0a45790a8b18bf1468c70957c55293e7fe32'
```

Comment:
You shouldn't expose your Github token, use 'dotenv' gem to hide this information.